src/components/AuthForm.js

```diff
@@ -39,6 +53,9 @@ function AuthForm (props) {
       case "Password":
         setPassword(data.value);
         break;
+
+      default:
```

Comment:
added a default case to the switch statement as it was creating a warning message.